### PR TITLE
MonitoringService.monitor() fails for unregistered

### PR DESCRIPTION
**Pull request by pirapira**
MonitoringService.

### CODE REVIEW

File: service_contracts.rst [hunk 1]
```diff
@@ -338,3 +338,11 @@ To handle the rewards, the MonitoringService contract provides two functions. ``
 
 .. autosolcontract:: MonitoringService
     :members: monitor, claimReward
+
+
+monitor()
+---------
+
+When the :ref:`MS` calls ``monitor()`` with a :term:`balance proof`, ``MonitoringService`` smart contract submits the balance proof on behalf of the :ref:`MS`.  If that succeeds, ``MonitoringSergvice`` smart contract records the fact that the :ref:`MS` has submitted the balance proof.
```
Comment (palango) [inline service_contracts.rst]: There's a typo in `MonitoringSergvice`

File: service_contracts.rst [hunk 2]
```diff
@@ -338,3 +338,11 @@ To handle the rewards, the MonitoringService contract provides two functions. ``
...
+
+* ``monitor()`` refuses callers that are not registered in ServiceRegistry.
```
Comment (palango) [inline service_contracts.rst]: Should `ServiceRegistry` be code styled as well?

File: service_contracts.rst (hunk 1 above)
Comment (pirapira) [reply service_contracts.rst]: Thank you.  I fixed this.

File: service_contracts.rst (hunk 2 above)
Comment (pirapira) [reply service_contracts.rst]: I fixed this as well.

**Pull request merged**